api-gateway/src/main/resources/application.properties

```
@@ -0,0 +1,27 @@
+spring.application.name=api-gateway
+server.port=8082
+eureka.client.service-url.defaultZone=http://localhost:8081/eureka
```

лучше передавать через переменные окружения с default значением
```suggestion
eureka.client.service-url.defaultZone=${EUREKA_DEFAULT_ZONE_URL:http://localhost:8081/eureka}
```